src/pages/DataSubmissionForm.js

```diff
@@ -1,15 +1,25 @@
 import { useState } from 'react';
-import {div, img} from 'react-hyperscript-helpers';
+import {h, div, img, h1, form} from 'react-hyperscript-helpers';
+import { set } from 'lodash/fp';
+
 import lockIcon from '../images/lock-icon.png';
 import {Styles} from '../libs/theme';
 
+import DataSubmissionStudyInformation from '../components/data_submission/ds_study_information';
+
 
 export const DataSubmissionForm = () => {
 
   // NOTE: remove after adding components
   /* eslint-disable no-unused-vars */
   const [formData, setFormData] = useState({});
 
+  const onChange = ({ key, value }) => {
+    /* eslint-disable no-console */
+    console.log('StudyInfo OnChange:', key, value);
```

Comment:
Nit: stray console.log

Reply:
i actually kind of liked it there... just while there was no other confirmation that the code was working as intended. Would you be okay with keeping this until the feature is completed?

Reply:
Sure, I was worried Codacy was going to freak out about it, but apparently not